ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/MultiTenancyPrivacyController.java

```diff
@@ -234,11 +234,7 @@ public boolean isGroupAdditionTransaction(final PrivateTransaction privateTransa
       final Bytes privacyGroupId, final String enclavePublicKey) {
     final Optional<PrivacyGroup> maybePrivacyGroup =
         privacyController.retrieveOnChainPrivacyGroup(privacyGroupId, enclavePublicKey);
```

Comment:
NIT: we can return the value directly instead of assigning it to `maybePrivacyGroup`.

Reply:
This is returning Optional because the off-chain equivalent does it ...

Reply:
Done